.github/workflows/build.yml

```diff
@@ -12,44 +12,68 @@ defaults:
 
 env:
   BUILD_TYPE: Release
+  WINDOWS_ARCH: x86
```

Comment:
Only builds 32-bit version? Unless I am missing something, I personally discourage it...

Reply:
There was demand for 32-bit Windows builds by at least one person. Do you want 64-bit builds-only instead, or 32-bit & 64-bit variants of the Windows builds?

Reply:
I'd prefer to have both.

Reply:
Done.